spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -1,51 +1,179 @@
 require 'spec_helper'
 
 describe ConferenceRegistrationsController, type: :controller do
-  let(:conference) { create(:conference) }
+  let(:conference) { create(:conference, title: 'My Conference', short_title: 'myconf') }
   let(:user) { create(:user) }
+  let(:not_registered_user) { create(:user) }
+  let(:registered_user) { create(:user) }
+  let!(:registration) { create(:registration, conference: conference, user: registered_user, created_at: 1.day.ago) }
+
+  shared_examples 'access action without ability' do |user, ichain, path, message|
```

Comment:
You call the shared example access action *without* ability, but you are using it for cases when the user has the ability to access the action. I would suggest we rename this. Perhaps even just `#new` ? What do you think?